x-pack/plugins/secops/public/components/load_more_table/index.mock.tsx

```diff
@@ -47,30 +46,34 @@ export const mockData = {
   },
 };
 
-export const getHostsColumns = (): Array<Columns<HostsEdges>> => [
+export const getHostsColumns = () => [
```

Comment:
The previous types were stronger than this new typing. With the previous typing, it was covering all of the types including `name, truncatedText`, etc... And didn't require individual types like below.

Should this be changed to:

`getHostsColumns = (): Array<Columns<string>> => [` if you want all strings rather than nodes being returned?